test/test_client.py

```diff
@@ -2,14 +2,14 @@
 from __future__ import unicode_literals
 
 import socket
+from test.helpers import producer_running, queue_to_list
```

Comment:
Could we add `known_local_folder = ["test"] to our isort config so that this gets correctly sorted down below?

Reply:
if we rename `test` to `tests` then we won't need to add that line because `tests` is the standard location for tests

pushed 2 commits which do that

first one renames the dir and the other one edits the files

Reply:
Ah good call!